UnicodeJsps/src/main/java/org/unicode/jsp/XPropertyFactory.java

```diff
@@ -145,48 +145,6 @@ public String transform(Integer source) {
                                 },
                                 false)
                         .setMain("toNFKD", "toNFKD", UnicodeProperty.STRING, "1.1"));
-
```

Comment:
I find these very useful in practice. Is there a compelling reason to remove?

Reply:
Well, they are very slow (getSet involves computing the preimage of a function).

But what do you use these for, as opposed to Lowercase_Mapping, Uppercase_Mapping, and Titlecase_Mapping?

Reply:
I just checked (on the staging JSPs which do not yet have these changes), and both of the following contain only characters whose mappings changed in 16.0 (because we are lagging on the ICU side used in the right-hand side):
* [:^Lowercase_Mapping=@toLowercase@:]
* [:^Uppercase_Mapping=@toUppercase@:]

[:^Titlecase_Mapping=@toTitlecase@:] contains one interesting difference, U+0345, because ICU skips past a leading U+0345 in titlecasing. But I think for all practical purposes, you can just use the mappings.

Reply:
Ok, makes sense.